test/java/seedu/addressbook/FormatterTest.java

```diff
@@ -40,32 +41,40 @@ public void formatPerson() throws Exception {
                 + " " + NEWLINE;
         assertEquals(expected, formatter.format(persons));
 
         //List of 3 people, all fields non-private
         persons = helper.generatePersonList(false, false , false);
-        expected = " \t1. Person 1 Phone: 1 Email: 1@email Address: House of 1   Tags: [tag1][tag2]" + NEWLINE
-                + " \t2. Person 2 Phone: 2 Email: 2@email Address: House of 2   Tags: [tag2][tag3]" + NEWLINE
-                + " \t3. Person 3 Phone: 3 Email: 3@email Address: House of 3   Tags: [tag4][tag3]" + NEWLINE
+        expected = " \t1. Person 1" + NEWLINE + " Phone: 1" + NEWLINE + " Email: 1@email" + NEWLINE
+                + " Address: House of 1" + NEWLINE + " " + NEWLINE + "  Tags: [tag1][tag2]" + NEWLINE
+                + " \t2. Person 2" + NEWLINE + " Phone: 2" + NEWLINE + " Email: 2@email" + NEWLINE
+                + " Address: House of 2" + NEWLINE + " " + NEWLINE + "  Tags: [tag2][tag3]" + NEWLINE
+                + " \t3. Person 3" + NEWLINE + " Phone: 3" + NEWLINE + " Email: 3@email" + NEWLINE
+                + " Address: House of 3" + NEWLINE + " " + NEWLINE + "  Tags: [tag4][tag3]" + NEWLINE
                 + " " + NEWLINE;
         assertEquals(expected, formatter.format(persons));
 
         //List of 3 people, all fields non-private*
         persons = helper.generatePersonList(false, true , false);
-        expected = " \t1. Person 1 Phone: 1 Email: 1@email Address: House of 1   Tags: [tag1][tag2]" + NEWLINE
-                + " \t2. Person 2      Tags: [tag2][tag3]" + NEWLINE
-                + " \t3. Person 3 Phone: 3 Email: 3@email Address: House of 3   Tags: [tag4][tag3]" + NEWLINE
+        expected = " \t1. Person 1" + NEWLINE + " Phone: 1" + NEWLINE + " Email: 1@email" + NEWLINE
+                + " Address: House of 1" + NEWLINE + " " + NEWLINE + "  Tags: [tag1][tag2]" + NEWLINE
+                + " \t2. Person 2" + NEWLINE + " " + NEWLINE + " " + NEWLINE
+                + " " + NEWLINE + " " + NEWLINE + "  Tags: [tag2][tag3]" + NEWLINE
+                + " \t3. Person 3" + NEWLINE + " Phone: 3" + NEWLINE + " Email: 3@email" + NEWLINE
+                + " Address: House of 3" + NEWLINE + " " + NEWLINE + "  Tags: [tag4][tag3]" + NEWLINE
                 + " " + NEWLINE;
         assertEquals(expected, formatter.format(persons));
     }
 
     @Test
     public void formatAllExam() throws Exception {
         TestDataHelper helper = new TestDataHelper();
-        List<Exam> exams = helper.generateExamList(false);
-        String expected = " \t1. Exam: Exam 1 Subject 1 01-02-2018 10:00 12:00 Held in 1" + NEWLINE
+        List<Exam> exams = helper.generateExamList(false, true);
+        String expected = " \t1. Exam: Exam 1 Subject 1 01-02-2018 10:00 12:00 Held in 1 0" + NEWLINE
+                + " \t2. private Exam: Exam 2 Subject 2 01-02-2018 10:00 12:00 Held in 2 0" + NEWLINE
                 + " " + NEWLINE;
         assertEquals(expected, formatter.formatExam(exams));
 
```

Comment:
Refer to previous comment on previous commit about using FormatStrings + Template Strings instead

Reply:
I left it as it is as we are still gonna work on the formatting?

Reply:
for the other tests above, there can be a standard formatter but seed 3 gives the tags as differently

Reply:
What I meant was using String.format(\t1. %s$1 \t2 %s$1, NEWLINE) or something like that to reduce the bloat of the code cause by "" and +